getting_started/configure_openshift.adoc

```diff
@@ -0,0 +1,237 @@
+installing a production environment of OpenShift.
+
+[[change-log-in-identity-provider]]
+== Change Log In Identity Provider
+
+The default behavior of a freshly installed {product-title} instance is to deny
+any user from logging in. To change the authentication method to HTPasswd:
+
+. Open the *_/etc/origin/master/master-config.yaml_* file in edit mode.
+. Find the `identityProviders` section.
+. Change
+`DenyAllPasswordIdentityProvider` to `HTPasswdPasswordIdentityProvider`
+provider.
+. Change the value of the name label to `htpasswd_auth` and add a
+new line `file: /etc/origin/openshift-passwd` in the provider section.
++
+An example `identityProviders` section with `HTPasswdPasswordIdentityProvider`
+would look like the following:
++
+----
+identityProviders: - challenge: true login: true
+name: htpasswd_auth provider: apiVersion: v1 file: /etc/origin/openshift-passwd
+kind: HTPasswdPasswordIdentityProvider
+----
+
+. Save the file.
+
+[[create-user-accounts]]
+== Create User Accounts
+
+Now that you are using the `HTPasswdPasswordIdentityProvider` provider, you
+need to generate these user accounts.
+
+. You can use the *httpd-tools* package to obtain the *htpasswd* binary that can
+generate these accounts.
++
+----
+yum -y install httpd-tools
+----
+
+. Create a user account.
++
+----
+touch /etc/origin/openshift-passwd htpasswd -b
+/etc/origin/openshift-passwd admin redhat
+----
++
+Note that you have created a
+user with the username of `admin` and password of `redhat`.
+
+. Restart OpenShift before going forward.
++
+----
+systemctl restart atomic-openshift-master
+----
+
+. Give this user account `cluster-admin` privileges (which allows it to do
+everything).
++
+----
+oadm policy add-cluster-role-to-user cluster-admin admin
+----
+
+. You can use this username/password combination to log in via the web
+console or the command line. To test this:
++
+----
+oc login -u admin
+----
+
+Before going forward, change to the `default` project.
+
+----
+oc project default
+----
+
+For more details, see
+xref:../admin_guide/manage_authorization_policy.adoc#admin-guide-manage-authorization-policy[roles]
+and
+xref:../install_config/configuring_authentication.adoc#install-config-configuring-authentication[authentication].
+
+[[deploy-router]]
+== Deploy the OpenShift Router
+
+The OpenShift router is the entry point for external network traffic destined
+for OpenShift services. It supports HTTP, HTTPS, and any TLS-enabled traffic
+that uses SNI, which enables the router to send traffic to the correct service.
+
+Without the router, OpenShift services and pods are unable to communicate with
+any resource outside of the OpenShift instance.
+
+The installer creates a default router.
+
+. Delete the default router using the following command:
++
+----
+oc delete all -l router=router
+----
+
+. Create a new default router:
++
+----
+$ oadm router --replicas=1 --service-account=router
+----
```

Comment:
Does it say anywhere in the docs why we delete the default and create a new one straight away? Is it just for this quick demo install? Should it say all this here? I'd imagine the reader would think this is weird (I do too).